src/librustdoc/doctest.rs

```diff
@@ -940,13 +940,14 @@ impl Tester for Collector {
                 let report_unused_externs = |uext| {
                     unused_externs.lock().unwrap().push(uext);
                 };
+                let option_no_run = options.no_run;
                 let res = run_test(
                     &test,
                     &cratename,
                     line,
                     options,
                     config.should_panic,
-                    config.no_run,
+                    config.no_run || option_no_run,
```

Comment:
```suggestion
                    config.no_run || options.no_run,
```
Why not just do this?

Reply:
The build failed when I try do to that.

>     Checking rustdoc v0.0.0 (/checkout/src/librustdoc)
> error[E0382]: use of moved value: `options`
>     |
> 947 |                     options,
>     |                     ------- value moved here
> 948 |                     config.should_panic,
> 948 |                     config.should_panic,
> 949 |                     config.no_run || options.no_run,
>     |                                      ^^^^^^^^^^^^^^ value used here after move
>     |
>     = note: move occurs because `options` has type `config::Options`, which does not implement the `Copy` trait
> error: aborting due to previous error
> 
> For more information about this error, try `rustc --explain E0382`.
> error: could not compile `rustdoc`
> ```

Reply:
What about `let no_run = config.no_run || options.no_run;`?

Reply:
seems good :) I am going to implement that.